snow/networking/handler/health.go

```diff
@@ -39,7 +42,7 @@ func (h *handler) HealthCheck(ctx context.Context) (interface{}, error) {
 
 func (h *handler) networkHealthCheck() (interface{}, error) {
```

Comment:
this is being run for every chain, so we might end up having same disconnected validator IDs for each chain.